static/app/views/issueDetails/streamline/eventNavigation.tsx

```diff
@@ -0,0 +1,371 @@
+                css={grayText}
+              />
+            </Navigation>
+            <LinkButton
+              to={{
+                pathname: normalizeUrl(
+                  `/organizations/${organization.slug}/issues/${group.id}/events/`
+                ),
+                query: omit(location.query, 'query'),
+              }}
+              borderless
+              size="xs"
+              css={grayText}
+            >
+              {t('View All Events')}
+            </LinkButton>
+          </NavigationWrapper>
+        </EventNavigationWrapper>
+        <Divider />
+        <EventInfoJumpToWrapper>
+          <EventInfo>
+            <EventIdInfo>
+              <EventTitle>{t('Event')}</EventTitle>
+              <Button
+                aria-label={t('Copy')}
+                borderless
+                onClick={copyEventId}
+                size="zero"
+                title={event.id}
+                tooltipProps={{overlayStyle: {maxWidth: 'max-content'}}}
+                translucentBorder
+              >
+                <EventId>
+                  {getShortEventId(event.id)}
+                  <CopyIconContainer>
+                    <IconCopy size="xs" />
+                  </CopyIconContainer>
+                </EventId>
+              </Button>
+              <DropdownMenu
+                triggerProps={{
+                  'aria-label': t('Event actions'),
+                  icon: <Chevron direction="down" color={theme.subText} />,
+                  size: 'zero',
+                  borderless: true,
+                  showChevron: false,
+                }}
+                position="bottom"
+                size="xs"
+                items={[
+                  {
+                    key: 'copy-event-id',
+                    label: t('Copy Event ID'),
+                    onAction: copyEventId,
+                  },
+                  {
+                    key: 'copy-event-link',
+                    label: t('Copy Event Link'),
+                    onAction: copyLink,
+                  },
+                  {
+                    key: 'view-json',
+                    label: t('View JSON'),
+                    onAction: downloadJson,
+                  },
+                ]}
+              />
+            </EventIdInfo>
+            <TimeSince date={event.dateCreated ?? event.dateReceived} css={grayText} />
+          </EventInfo>
+          <JumpTo>
+            <div>{t('Jump to:')}</div>
+            <ButtonBar>
+              {jumpToSections.map(jump => (
+                <Button
+                  key={jump.section}
+                  onClick={() => {
+                    document
+                      .getElementById(jump.section)
+                      ?.scrollIntoView({block: 'start', behavior: 'smooth'});
+                  }}
+                  borderless
+                  size="xs"
+                  css={grayText}
+                >
+                  {jump.label}
+                </Button>
```

Comment:
i feel like we could still make these anchor links? that way we don't lose the ability to share an anchor link to a specific section

Reply:
So the anchor links should still work since the sections are ID'd properly, but it can't smooth scroll if we just use `href` to `#section`. Is that still preferred?

Reply:
Also it looks like the navigation is a little buggy, it seems to dismiss the `#anchor` refs in the URL and clicking 'next event' a few times eventually just loads recommended again 👀  probably something to address on its own.